docs(atomic-angular): add explanation on initialization process (#1785)
coveo/ui-kit: packages/atomic-angular/README.md

Once this is done, you will be able to reference all atomic components inside that module.

> ## Initializing the search interface

Reviewer: ```suggestion
## Initializing the Search Interface
```

> You can initialize the search interface at any time you desire in the lifecycle of your application. One such suitable lifecyle hook is the `AfterViewInit`,

Reviewer: ```suggestion
You can initialize the search interface at any time in the lifecycle of your application. One such suitable lifecyle hook is `AfterViewInit`.
```

> For example, using the above `AppModule` as a starting point, which bootstrap `AppComponent`:

Reviewer: Oh I think I see now, thanks!
```suggestion
The following example uses this `AfterViewInit` hook, and relies on the above `AppModule` as a starting point, which bootstraps `AppComponent`.
```

> For example, using the above `AppModule` as a starting point, which bootstrap `AppComponnent`:

Reviewer: I don't really see what you mean here: "which bootstrap `AppComponnent`" ?

Author: If you look at the full README, in the `AppModule` code, there's a `bootstrap` configuration
